### PR TITLE
feat- added login/singup pages

### PR DESCRIPTION
**Pull request by Balakumar3018**
responsive yet to add. click create account for signup page

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -1,16 +1,22 @@
 @import url("https://orangeui.netlify.app/orange-ui.css");
+body{
+    background-color: white;
+}
 .search-bar2{
     margin-top: -10px;
 }
 .navigation-bar{
     /* border: none; */
     height: 60px;
```
Comment (harshitpaliwal95) [inline main.css:10]: use rems

**Review by harshitpaliwal95 — APPROVED** [review]
- ui looks good

**Review by Koushith — APPROVED** [review]
Try using relative units, rest everything looks good to me.

File: main.css [hunk 2]
```diff
@@ -1,16 +1,22 @@
...
-    width: 100%;
+    /* width: 100%; */
 }
 .btn-login{
     height: 30px;
     margin-top: 20px;
 }
+#search-bar{
+    height: 30px;
```
Comment (prath-dev) [inline main.css:18]: Always use rem avoid using px ...rem is also good for responsive use px upto 10px only and also avoid using id to select element always use classes ..

**Pull request merged**